lib/mixins/template-stamp.html

```diff
@@ -313,6 +315,32 @@
        *   metadata to `nodeInfo`
        */
       static _parseTemplateNestedTemplate(node, outerTemplateInfo, nodeInfo) {
+        // If a `<dom-repeat>` or a `<dom-if>` is inside of an svg, we need
+        // to create all elements in a different namespace. To do so, make an svg,
+        // let the HTML parser parse in that namespace, and then put back all
+        // created children in a fresh template.
+        if (node.parentNode.localName && templateExtensions[node.parentNode.localName]
+              && node.parentNode.parentNode && node.parentNode.parentNode.namespaceURI === SVG_NAMESPACE) {
+          const svg = document.createElementNS(SVG_NAMESPACE, 'svg');
+          svg.innerHTML = node.innerHTML;
+
+          const template = /** @type {HTMLTemplateElement} */ (document.createElement('template'));
+
+          while (svg.firstChild) {
+            template.content.appendChild(svg.removeChild(svg.firstChild));
+          }
+
+          node.parentNode.replaceChild(template, node);
+
+          // Clean up, such that `_parseTemplateChildNodes` does not run again
+          // on the old template children
```

Comment:
Why would this have been the case?

Reply:
The children of template are not placed in a documentfragment (see previous reply) and are therefore direct children. Since parsetemplatenode still holds a reference to the original "template", it will iterate through the "real" children. But thats wrong, because we expect it to be in content instead.